alert-database/src/main/java/com/synopsys/integration/alert/database/api/DefaultNotificationAccessor.java

```diff
@@ -185,6 +174,27 @@ public PageRequest getPageRequestForNotifications(Integer pageNumber, Integer pa
         return PageRequest.of(pageNumber, pageSize, Sort.by(sortingOrder));
     }
 
+    @Override
+    public AlertPagedModel<AlertNotificationModel> getFirstPageOfNotificationsNotProcessed() {
```

Comment:
It seems like we should be passing the `pageSize` as a parameter. This class shouldn't be the one that decides on that size.